runtime/doc/eval.txt

```diff
@@ -6360,7 +6360,7 @@ synstack({lnum}, {col})					*synstack()*
 		valid positions.
 
 system({cmd} [, {input}])				*system()* *E677*
-		Get the output of the shell command {cmd} as a |string|.  {cmd}
+		Get the output of the command {cmd} as a |string|.  {cmd}
 		will be run the same as in |jobstart()|.  See |systemlist()|
 		to get the output as a |List|.
```

Comment:
Below this it mentions:

> When {input} is given and is a string this string is written 
> to a file and passed as stdin to the command. 

But `do_os_system()` looks like it sends the input directly to the process.